code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,318 @@
+	set desc = "List all current handling ahelps"
+
+	if(!check_rights(R_BAN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	var/count = 0
+
+	for(var/datum/adminticket/T in admintickets)
+		if(T.resolved == "No" && T.admin == src.ckey)
+			count++
+
+	if(count < 1)
+		usr << "<b>You don't have any ACTIVE ahelps!</b>"
+		return
+
+
+
+	for(var/datum/adminticket/T in admintickets)
+		if(T.resolved == "No" && T.admin == src.ckey)
+			var/ref_mob = "\ref[T.mob]"
+			usr << "<span class='adminnotice'><b><font color=red>#[T.ID] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</b></A><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+			usr << "	<b>Controls:</b> (<A HREF='?_src_=holder;adminmoreinfo=[ref_mob]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=[ref_mob]'>PP</A>) (<A HREF='?_src_=vars;Vars=[ref_mob]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=[ref_mob]'>SM</A>) (<A HREF='?_src_=holder;traitor=[ref_mob]'>TP</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=[ref_mob]'>FLW</A>)"
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			if(T.resolved == "No")
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+			else
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+/client/verb/viewmyahelp()
+	set category = "Admin"
+	set name = "View my Ahelps"
+	set desc = "List your ahelps"
+
+	var/count = 0
+
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey)
+			count++
+
+	if(count < 1)
+		usr << "<b>You don't have any ahelps!</b>"
+		return
+
+	usr << "<b>Resolved Ahelps</b>"
+
+	var/rpass = 0
+
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "Yes")
+			rpass = 1
+			usr << "<span class='adminnotice'><b><font color=red>Adminhelp ID: #[T.ID] </font></b></class>"
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			usr << "	<b>Resolved:</b> [T.resolved]"
+
+	if(rpass == 0)
+		usr << "	None"
+
+	usr << "<b>Unresolved Ahelps</b>"
+
+	var/upass = 0
+
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
+			upass = 1
+			usr << "<span class='adminnotice'><b><font color=red>Adminhelp ID: #[T.ID] </font></b></class>"
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			usr << "	<b>Resolved:</b> [T.resolved]"
+
+	if(upass == 0)
+		usr << "	None"
+
+/client/proc/createticket(var/player, var/message, var/uckey, var/mob)
```

Comment:
can you be more descriptive with those arguments?what is uckey referred to and what is the mob var referring to,usually?